pandas/core/strings.py

```diff
@@ -854,8 +854,8 @@ def str_extract(arr, pat, flags=0, expand=True):
     pat : string
         Regular expression pattern with capturing groups.
     flags : int, default 0 (no flags)
-        ``re`` module flags, e.g. ``re.IGNORECASE``.
-        See :mod:`re`
+        One of the ``re`` module flags, e.g. ``re.IGNORECASE``.
+        See :mod:`re`.
```

Comment:
Do you mind briefly explaining what these flags are about. Nothing long, but something a bit more descriptive than what we've got now just linking to the `re` documentation. Something like "Flags that modify regular expression matching for things like case, spaces, etc."